core-plugins/src/main/java/co/cask/hydrator/plugin/batch/source/AzureBatchSource.java

```diff
@@ -61,7 +61,6 @@ public AzureBatchSource(AzureBatchConfig config) {
     @Override
     protected Map<String, String> getFileSystemProperties() {
       Map<String, String> properties = new HashMap<>(super.getFileSystemProperties());
-      properties.put("fs.defaultFS", String.format("wasb://%s@%s/", container, account));
```

Comment:
so we can add container and corresponding account while accessing the files, similar to S3 source plugin. For example: